### PR TITLE
Revert "Upgrade deps (#569)"

### PR DESCRIPTION
**Pull request by tholonious**
This reverts commit 9f108080ed8b13d8b0f7907539fc942e96fb32f3.

### CODE REVIEW

**Comment by tholonious** [comment]
* **#630** <a href="https://app.graphite.dev/github/pr/deso-protocol/backend/630?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `feature/proof-of-stake`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tholonious and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: routes/dao_coin_exchange.go [hunk 1]
```diff
@@ -688,9 +689,9 @@ func CalculateScaledExchangeRateFromPriceString(
 	// Beyond this point, we know that the operation type is lib.DAOCoinLimitOrderOperationTypeBID
 
 	// Scale up the price to account for DAO Coin -> DESO trades
-	if IsDesoPkid(buyingCoinPublicKeyBase58Check) {
```
Comment (lazynina) [inline routes/dao_coin_exchange.go:691]: I think there was some other reason DH wanted this change. I think it matches both DESO and the zero public key

File: routes/dao_coin_exchange.go [hunk 2]
```diff
@@ -1106,14 +1107,14 @@ func (fes *APIServer) validateTransactorSellingCoinBalance(
 	sellingCoinPKID := &lib.ZeroPKID
 
 	// Calculate current balance for transactor.
-	transactorSellingBalanceBaseUnits := uint256.NewInt(0)
-	if IsDesoPkid(sellingDAOCoinCreatorPublicKeyBase58Check) {
+	transactorSellingBalanceBaseUnits := uint256.NewInt()
+	if sellingDAOCoinCreatorPublicKeyBase58Check == DESOCoinIdentifierString {
```
Comment (lazynina) [inline routes/dao_coin_exchange.go:1111]: Same here with the IsDESOPKID

File: routes/dao_coin_exchange.go [hunk 3]
```diff
@@ -1336,8 +1337,8 @@ func (fes *APIServer) getTransactorDesoOrDaoCoinBalance(
 }
 
 func getScalingFactorForCoin(coinCreatorPublicKeyBase58Check string) *uint256.Int {
-	if IsDesoPkid(coinCreatorPublicKeyBase58Check) {
```
Comment (lazynina) [inline routes/dao_coin_exchange.go:1339]: IsDesoPkid should stay I think

**Review by lazynina — CHANGES_REQUESTED** [review]

**Review by lazynina — APPROVED** [review]

**Pull request merged**